doc/deployment-on-production-env.md

```diff
@@ -0,0 +1,43 @@
+# Deployment on Production Environment
+
+In this guide we will cover installation of pre-requisites on your 
+production cluster.
+
+# Requirements
+
+## A Cluster
+
+This guide requires you to have your own Kubernetes/OpenShift cluster.
+
+## Install Supported Multi-Network CNI
+
+Install any supported [Network Plumbing Working Group de-facto standard](https://github.com/k8snetworkplumbingwg/multi-net-spec) implementation.
+For example - [Multus](https://github.com/intel/multus-cni). To deploy multus on a kubernetes cluster with flannel cni:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/hack/multus/kubernetes-multus.yaml
+kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/hack/multus/multus.yaml
+```
+
+## Install CNI plugins
+
+* [CNI plugins](https://github.com/containernetworking/plugins) must be installed in the cluster, to allow collaboration with 
```

Comment:
I would omit this installation guide too. Let's talk about it as a prerequisite only - have CNI plugin that supports MAC configuration.

Reply:
not needed once I removed the production guide. will remove